tools/ShapeChecker/src/main/java/net/open_services/scheck/shapechecker/HttpHandler.java

```diff
@@ -143,9 +150,32 @@ else if (containsMatch(skipURIPatterns,httpUri.toString()))
     }
 
 
+    /**
+     * Hack to work around issue with Jena handling of Dublin Core redirects.
+     * @param httpUriOrig a URI
+     * @return if the input was a DCTerms URI, return a modified URI reflecting the several redirects
+     */
+    private static URI issue88(URI httpUriOrig)
```

Comment:
Maybe call it dctermsRedirectWorkaround? You can link the issue in the javadoc.

Reply:
OK - I'll change this.